templates/download/iot/intel-iotg.html

```diff
@@ -0,0 +1,71 @@
+{% extends "download/_base_download.html" %}
+
+{% block title %}Download Ubuntu for Intel IoT platforms | Download{% endblock %}
+{% block meta_description %}​​Canonical and Intel have partnered to provide Ubuntu Desktop and Ubuntu Core on Intel's IoT platforms: Atom&reg; X6000E Series, Intel&reg; Pentium&reg; and Celeron&reg; N and J Series and 11th Gen Intel&reg; Core&trade; processors.{% endblock meta_description %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1hxKzPs6e6JQaa9tBqfZ55GAXrVnEryEWBja8HlXy-lc/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row u-vertically-center">
+    <div class="col-7">
+      <h1>Download Ubuntu for Intel IoT platforms</h1>
+      <p class="p-heading--4">Ubuntu - now optimized for Intel's IOTG platforms</p>
+      <p>Ubuntu Core 20 and Ubuntu Desktop 20.04 based images for Elkhart Lake and Tiger Lake platforms will be released on October 14th and are currently available as a private alpha.</p>
+      <a class="p-button--positive" href="mailto:sales-iotg@canonical.com&subject=Ubuntu+Core+20+private+alpha+for+Elkhart+Lake+and+Tiger+Lake">Contact us if you'd like early access</a>
```

Comment:
I hate this, it's an anti-pattern these days. [:information_source:](https://blog.seibert-media.com/2014/10/06/why-mailto-links-should-be-avoided-on-websites/)

Reply:
I agree - Peter added this yesterday I presumed it was a placeholder due to the urgency of this page. (The comment you left on the copy doc has been resolved by David, though I'm not sure why as it hasn't been answered)